test/indexer.py

```diff
@@ -29,7 +29,7 @@ def init_elasticsearch():
   # Wait for Elasticsearch to come up.
   # I had trouble getting wait_for_status to work, so just use sleep. See
   # https://discuss.elastic.co/t/cant-get-python-client-wait-for-status-to-work/123163
-  time.sleep(10)
+  time.sleep(20)
```

Comment:
Can we do 15? Then we won't have to wait as long until we can use UI.

Reply:
Playing it dangerous but sure